src/compiler/Restler.Compiler/SwaggerSpecPreprocessor.fs

```diff
@@ -175,7 +175,17 @@ let rec private inlineFileRefs2
                                             if ref.Count() > 0 then
                                                 ref
                                             else
-                                                obj.Properties()
+                                                // TODO: Temporary workaround for issue #61 - NSWag failure to parse
+                                                // required boolean in Headers.  Remove this when the NSWag bug is fixed.
+                                                // The workaround is to remove the required property.
+                                                // This is fine for now, since RESTler does not currently fuzz or
+                                                // learn from header values.
+                                                let headerInPath = sprintf "%s.%s" "headers" x.Name
+                                                if obj.Path.EndsWith(headerInPath) then
+                                                    //let headerName = obj. = obj.Path.Substring("headers." + "headers.".Length)
```

Comment:
Is this comment intentional?

Reply:
No, this needs to be removed. Thanks for catching

---
In reply to: [528958954](https://github.com/microsoft/restler-fuzzer/pull/86#discussion_r528958954) [](ancestors = 528958954)